catalog/glue/glue_test.go

```diff
@@ -778,12 +781,103 @@ func TestGlueListNamespacesIntegration(t *testing.T) {
 	}
 	assert := require.New(t)
 
-	awscfg, err := config.LoadDefaultConfig(context.TODO(), config.WithClientLogMode(aws.LogRequest|aws.LogResponse))
+	awsCfg, err := config.LoadDefaultConfig(context.TODO(), config.WithClientLogMode(aws.LogRequest|aws.LogResponse))
 	assert.NoError(err)
 
-	catalog := NewCatalog(WithAwsConfig(awscfg))
+	ctlg := NewCatalog(WithAwsConfig(awsCfg))
 
-	namespaces, err := catalog.ListNamespaces(context.TODO(), nil)
+	namespaces, err := ctlg.ListNamespaces(context.TODO(), nil)
 	assert.NoError(err)
 	assert.Contains(namespaces, []string{os.Getenv("TEST_DATABASE_NAME")})
 }
+
+func TestGlueCreateTableSuccessIntegration(t *testing.T) {
+	if os.Getenv("TEST_DATABASE_NAME") == "" {
+		t.Skip()
+	}
+	if os.Getenv("TEST_TABLE_NAME") == "" {
+		t.Skip()
+	}
+	if os.Getenv("TEST_TABLE_LOCATION") == "" {
+		t.Skip()
+	}
```

Comment:
Is there any known test framework we can use to test Glue in CI outside of actually having to use AWS directly?

Reply:
I'd have to look deeper at how glue works in order to figure out the difference, I'll get back to you

Reply:
I'm very sorry I must have overlooked this :cold_sweat: . For the other test case `TestGlueRenameTable` I think you are skipping over it with a note
https://github.com/apache/iceberg-go/blob/16e6b69db76b25146e3c286c6c95be2410d2cc89/catalog/glue/glue_test.go#L601
https://github.com/apache/iceberg-go/blob/16e6b69db76b25146e3c286c6c95be2410d2cc89/catalog/glue/glue.go#L213-L217

I have also added `TestGlueCreateTableRollbackOnInvalidMetadata`. Please let me know if you want to add a similar test case for the happy path and skip over it until we have addressed your comment in `TODO`